exercises/bank-account/uBankAccountTests.pas

```diff
@@ -37,16 +37,19 @@ implementation
 
 procedure BankAccountTests.Returns_empty_balance_after_opening;
 var account: IBankAccount;
+    Expected: double;
 begin
+  Expected := 0.00;
```

Comment:
It isn't technically needed.  I was keeping in the spirit of the banking theme.  :smile:  I had a student submit a solution that passed all the tests but wasn't really thread safe.  They had utilized an object I wasn't aware of to attempt thread safe writing to the bank balance.  The mechanism they used utilized integers and so their solution utilized integers instead of a float type.  I think the exercise might be improved if some cents were added to it.  

Their [1st](http://exercism.io/submissions/133ecbcfad4a486cb9684884fb9b4583) and [2nd](http://exercism.io/submissions/47f7ca82502f4784b9a774b9c4ec57ad) iterations.